### PR TITLE
Subscribers paginated and bundeled demographic data

### PR DESCRIPTION
**Pull request by kaspernj**
Asana: https://app.asana.com/0/7005399303210/16381172566150
Task: https://tasks.kaspernj.org/tasks/1404

Still some work to be done (think I blocked the queue, and I must wait for it to finish before continuing).

- [x] Check that fields-argument actually work.
- [x] Check that paginated subscribers actually work.

Various work on paginated subscribers to spare memory and bundle demographic data to avoid fetching it one by one.

Fixed method definition standards.
Custom timeout for subscribers (this request might take a long time to generate for Apsis).

### CODE REVIEW

File: spec/apsis-on-steroids_spec.rb [hunk 1]
```diff
@@ -110,12 +110,48 @@
...
+      count.should > 0
+    end
+
+    it "should get subscribers paginated" do
+      original_sub = sub
+
+      count = 0
+      mlist.subscribers_paginated do |sub_i|
+        count += 1
+        #puts "Subscriber: #{sub_i}"
```
Comment (kaspernj) [inline spec/apsis-on-steroids_spec.rb]: @nj: Fixed in 756294e.

File: include/mailing_list.rb [hunk 2]
```diff
@@ -29,16 +29,31 @@ def create_subscribers(data)
   end
 
   # Returns the subscribers of the mailing list.
-  def subscribers
+  SUBSCRIBERS_VALID_ARGS = [:all_demographics, :timeout, :field_names, :allow_paginated]
```
Comment (kaspernj) [inline include/mailing_list.rb]: @nj: Fixed in 6f67779.

File: README.md [hunk 3]
```diff
@@ -98,6 +98,20 @@ aos.subscribers do |sub|
 end
 ```
 
+### Bundle various field names in same request
+```ruby
+aos.subscribers(field_names: ["MyData1", "MyData2"]) do |sub|
```
Comment (kaspernj) [inline README.md]: @nj: Added in c51843f.

**Comment by nj** [comment]
:+1:

**Pull request merged**